app/client/ui/ActiveUserList.ts

```diff
@@ -58,20 +92,19 @@ function createUserIndicator(user: Partial<FullUser>, options = { overlapLeft: f
   );
 }
 
-function createRemainingUsersIndicator(users: Partial<FullUser>[], userCount?: number) {
-  const count = userCount ?? users.length;
+function createRemainingUsersIndicator(usersObs: Observable<Partial<FullUser>[]>, userCountObs?: Observable<number>) {
   return cssRemainingUsersButton(
     cssRemainingUsersImage(
-      `+${count}`,
+      domComputed(use => `+${userCountObs ? use(userCountObs) : use(usersObs).length}`),
       cssUserImage.cls("-medium"),
       dom.style("font-size", "12px"),
     ),
     menu(
-      () => users.map(user => remainingUsersMenuItem(
+      () => domComputed(usersObs, users => users.map(user => remainingUsersMenuItem(
```

Comment:
```suggestion
      () => [dom.forEach(usersObs, user => remainingUsersMenuItem(
```

Reply:
Weird. Did you wrap the `dom.forEach` in square brackets? My suggestion only includes the opening bracket because of limitations in GitHub's UI.